jscomp/bsb_exe/rescript_main.ml

```diff
@@ -135,10 +115,10 @@ let build_subcommand ~start argv argv_len =
     ~argv
     [|
       ("-w", unit_set_spec watch_mode, "Watch mode");
-      ("-with-deps", unit_set_spec make_world, "Build dependencies explicitly");
+      ("-with-deps", unit_set_spec (ref true), "*deprecated* Build dependencies explicitly");
```

Comment:
Might be misleading.
It's deprecated because redundant and it's unclear from the message.

Reply:
True. Or should we remove it completely?

Reply:
I'd say let's wait until v12 with removing it so that we don't break people's scripts.

Maybe the message could say something like

```
*deprecated* This is the default behavior now. This option will be removed in a future release.
```